src/ensembl/src/content/home/Home.tsx

```diff
@@ -15,25 +15,160 @@
+                      <span>Species selector</span>
+                      <SpeciesSelectorIcon className={styles.icon} />
+                    </div>
+                  </Link>
+                </div>
+                <div className={styles.bottomTextContainer}>
+                  <div className={styles.bottomText}>
+                    Create & manage your own species list
+                  </div>
+                </div>
+              </div>
+
+              <div className={styles.selectionContainer}>
+                <div className={styles.buttonsContainer}>
+                  <Link to="/genome-browser">
+                    <div className={styles.button}>
+                      <span>Genome browser</span>
+                      <BrowserIcon className={styles.icon} />
+                    </div>
+                  </Link>
+                </div>
+                <div className={styles.bottomTextContainer}>
+                  <div className={styles.bottomText}>
+                    Look at genes & transcripts in their genomic context
+                  </div>
+                </div>
+              </div>
+
+              <div className={styles.selectionContainer}>
+                <div className={styles.buttonsContainer}>
+                  <Link to="/entity-viewer">
+                    <div className={styles.button}>
+                      <span>Entity viewer</span>
+                      <EntityViewerIcon className={styles.icon} />
+                    </div>
+                  </Link>
+                </div>
+                <div className={styles.bottomTextContainer}>
+                  <div className={styles.bottomText}>
+                    Get gene & transcript information
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          {isEnvironment([Environment.DEVELOPMENT, Environment.INTERNAL]) ? (
+            <div className={styles.homeTopRight}>
+              <Link to="/help">{helpButton}</Link>
+            </div>
+          ) : (
+            <div
+              className={classNames(
+                styles.homeTopRight,
+                styles.homeTopRightDisabled
+              )}
+            >
+              {helpButton}
+            </div>
+          )}
+        </div>
+
+        <div className={styles.speciesContainer}>
+          <span className={styles.platform}></span>
+          <img src={speciesStripUrl} className={styles.speciesStrip} />
+        </div>
+      </div>
+      <div className={styles.footer}>
+        <div className={styles.footerLeft}>
+          <Logotype className={styles.logotype} />
+          <img src={ebiLogoUrl} className={styles.ebiLogo} />
+          <div className={styles.text}>
+            Wellcome Genome Campus, Hinxton, Cambridgeshire CB10 1SD, UK
+          </div>
+        </div>
+
+        <div className={styles.footerRightIcons}>
+          <a
+            href="//www.ensembl.info"
```

Comment:
Here and below: there is no need to depend on the protocol over which our page is served when deciding which protocol to use for an external site. All these links should explicitly use `https://`

Reply:
Done